api/nakadi-event-bus-api.yaml

```diff
@@ -1083,6 +1131,35 @@ definitions:
         items:
           $ref: '#/definitions/Event'
 
+  Subscription:
```

Comment:
@whiskeysierra Probably you are trying to find complete specification and it's not ready still.
the WIP version is in https://github.com/zalando/nakadi/tree/feature/ARUHA-190-subscription-api